airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/JdbcBufferedConsumerFactory.java

```diff
@@ -80,7 +80,7 @@ public static SerializedAirbyteMessageConsumer createAsync(final Consumer<Airbyt
         catalog,
         new BufferManager((long) (Runtime.getRuntime().maxMemory() * 0.2)),
         defaultNamespace,
-        Executors.newFixedThreadPool(2));
+        Executors.newFixedThreadPool(1));
```

Comment:
How do you feel about using 10% memory factor and 2 threads ?

Reply:
:shrug: I'll kick off a sync with the current branch and let it run overnight and we can reevaluate tomorrow morning I guess

Reply:
first attempt (with 0.2x factor + 1 thread) still got killed. Second attempt succeeded in 3 hours

I'll try 0.1x factor I guess :/ but I think that means we can't reliably go to 2 threads

Reply:
... and I guess I can try enabling the string dedup jvm option later, too 🤔 need to find where to actually inject the jvm flags though